subsys/usb/Kconfig

```diff
@@ -79,6 +79,12 @@ config USB_DEVICE_REMOTE_WAKEUP
 	help
 	  This option requires USBD peripheral driver to also support remote wakeup.
 
+config USB_DEVICE_DRIVER_HANDLE_CTRL_EPIN_ZLP
```

Comment:
Shouldn't you be enabling this by default in Nordic ICs?

Reply:
:man_facepalming: forgot `git add drivers`